frontend/src/utils/sessionStateOperations/index.ts

```diff
@@ -0,0 +1,4 @@
+import sortByDisplayPriority from './sortByDisplayPriority';
+import togglePinAndSortByDisplayOrder from './togglePinAndSortByDisplayOrder';
+
+export { sortByDisplayPriority, togglePinAndSortByDisplayOrder };
```

Comment:
I'm not asking for changes, but I feel like these could be class methods for a `SubscriberWrapper` class.

Reply:
hmm not really since subscriberWrapper is the base object, these operations are on the session level array of subscriberWrappers.

Reply:
Let's look into where to put this when we refactor the session stuff 🙏

Reply:
You're right, it feels like they could be grouped together in some way. Sorry for the noise